pages/common/mvn.md

```diff
@@ -0,0 +1,17 @@
+# mvn
+
+> Runnign Apache Maven.
+
+- Most common usage is to invoke a life cicle phase:
+
+`mvn package`
+`mvn install`
+`mvn deploy`
+
+- Invoke more that one phase with arguments:
+
+`mvn clean -P a_profile package clean`
+
+- Running an spring boot project with remote debug:
+
+`mvn spring-boot:run -Drun.jvmArguments="-Xdebug -Xrunjdwp:transport=dt_socket,server=y,suspend=n,address=5005"`
```

Comment:
`spring-boot:run` is pretty esoteric. A more common maven usage would be to run a main class in your project:

----
- Run a class with a main method:

`mvn exec:java -Dexec.mainClass="{{com.example.Main}}" -Dexec.args="{{arg1 arg2}}"`